gui/module.mk

```diff
@@ -32,6 +32,13 @@ MODULE_OBJS := \
 	widgets/scrollbar.o \
 	widgets/tab.o
 
+# Even if it seems redundant, please keep
+# these directives in that order!
+# This is needed by the "create_project" tool, for the OS X / iOS Xcode project
```

Comment:
I don't understand how this works. create_project seems to set both `IPHONE` and `MACOSX` when going over our files. Now, the comment in [here](https://github.com/bSr43/scummvm/blob/ios-fix/devtools/create_project/create_project.cpp#L343-L347) suggests that create_project selects based on the suffix. But, if IPHONE is defined only `browser.o` is ever picked up?

At any rate, this deserves a `HACK` (and probably `FIXME`) indication.